src/main/kotlin/com/coder/jetbrains/services/CoderPortForwardService.kt

```diff
@@ -48,12 +56,44 @@ class CoderPortForwardService(
     }
 
     private fun start() {
-        logger.info("starting port scanner")
+        // TODO: make path configurable?
+        val devcontainerFile = File(System.getProperty("user.home"), ".cache/JetBrains/devcontainer.json")
```

Comment:
I originally wanted to make this a setting, but this turns out to be a bit complicated because this plugin doesn't have any settings yet, so there's a bit of infrastructure involved. I can come back to this if needed, or we can go with this fixed path for now (or some other path). It's pretty straightforward for other software on the workspace to put the file in the expected place or create a symlink.

Reply:
Moved to a settings class, please let me know if the location/naming makes sense.